andino_firmware/src/hw.h

```diff
@@ -29,17 +29,17 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #pragma once
 
-// PC4 (pin A4), RIGHT ENCODER PIN A
-#define RIGHT_ENCODER_A_GPIO_PIN PC4
+// PC4 (pin 18), RIGHT ENCODER PIN A
```

Comment:
For a simpler matching of the arduino pin probably makes sense to also leave the A4 nomenclature here. Wdyt?

Reply:
Totally! For consistency, I will improve all pinout comments on a follow-up PR (to prevent missing approval here). Thanks!